scripts/fetch.sh

```diff
@@ -32,7 +32,7 @@ process_ref() {
   fi
 
   # Copy only curated subset of files from that repo into a tmp module dir.
-  rsync -amR --include-from="${module_static_path}/rsync.incl" . "${mod_tmp_path}"
+  rsync -amRL --include-from="${module_static_path}/rsync.incl" . "${mod_tmp_path}"
```

Comment:
`-L` copies symlinks as files, which is the case for this source repo, linking `LICENSE` from the repo root.

Reply:
We might want to expand to use long options here - i.e. `--copy-links` instead of `-L`.